package.json

```diff
@@ -50,8 +50,10 @@
     "@types/lodash.startswith": "^4.2.3",
     "@types/minimatch": "^3.0.1",
     "@types/node": "^8.0.26",
+    "@types/resolve": "0.0.4",
     "@types/webpack": "^3.0.10",
     "chai": "^3.5.0",
+    "css-loader": "^0.28.7",
```

Comment:
do we need css-loader here?

Reply:
Yeah, `css-loader` is for the Vue integration test. Since Vue uses single-file-components, I wanted to include a simple css section in the example files just to ensure everything is working. Also, I added an `import exampleCss from "./example.css"` to ensure other "non-vue" imports work (as reported by @aweiu @CKGrafico @jvbianchi).